libcalico-go/lib/validator/v3/validator.go

```diff
@@ -45,8 +45,9 @@ const (
 	// Maximum size of annotations.
 	totalAnnotationSizeLimitB int64 = 256 * (1 << 10) // 256 kB
 
-	// linux can support route-tables with indices up to 0xfffffff
-	routeTableMaxLinux uint32 = 0xffffffff
+	// linux can support route-tables with indices up to 0xfffffff, however, using all of them would likely blow up, so cap the limit at 65535
```

Comment:
Two thoughts about this note.
1. It's not clear what "blow up" means or what's doing the up-blowing. Perhaps: slow things down, cause a panic, melt the CPU ...
2. By 'cap the limit' do you mean:
- I, the note writer, recommend that you cap the limit. (But it's not capped by default.)
- I, the note writer, capped the limit, and I'm explaining why. (But you can use the linux max if you want.)